dataStore test runs properly only once

It seems that dataStore tests do not delete from database...

`npm test`
First time: `14 passing (167ms)`
Second time: `8 passing (180ms) 6 failing`
All errors are coming from `DB with redis backend` and are similar to:

```
 1) DB with redis backend with 0 completed crawls getHistoricalCrawlData should return empty list:
      AssertionError: expected 1 to equal 0
      + expected - actual
      -1
      +0
```

--- a/dataStore/backend-redis.js
+++ b/dataStore/backend-redis.js
@@ -109,10 +109,19 @@ function configure(opts) {
   });
 }
 
+function flush() {
+  return new Promise(resolve => {
+    client.flushdb(() => {
+      resolve();
+    });
+  });
+}
+
 export default {
   getHistoricalCrawlData,
   getHistoricalURLData,
   updateWithCurrentCrawlResult,
   finishCurrentCrawl,
   configure,
+  flush,
 };

--- a/dataStore/dataStore.js
+++ b/dataStore/dataStore.js
@@ -42,8 +42,15 @@ function configure(configOpts) {
   return Promise.resolve();
 }
 
+function flush() {
+  if (backend.flush) {
+    backend.flush();
+  }
+}
+
 const exported = {
   configure,
+  flush,
 };
 
 expectedFunctions.forEach((expectedFn) => {

--- a/test/dataStore.js
+++ b/test/dataStore.js
@@ -238,5 +238,9 @@ describe('DB with redis backend', () => {
     });
   });
 
+  after(() => {
+    return dataStore.flush();
+  });
+
   doTests();
 });

--- a/test/dataStore.js
+++ b/test/dataStore.js
@@ -238,9 +238,7 @@ describe('DB with redis backend', () => {
     });
   });
 
-  after(() => {
-    return dataStore.flush();
-  });
+  after(() => dataStore.flush());
 
   doTests();
 });

